### ISSUE TITLE
Cannot reproduce SSV2 finetuning results

### ISSUE THREAD

**Issue by wren93** [wgcban/adamae#1]
Hi, thank you for this great work. I'm trying to reproduce the results for SSV2 pretraining -> SSV2 fine-tuning using the checkpoint provided in the repo. I tried to follow this script https://github.com/wgcban/adamae/blob/main/scripts/kinetics/videomae_vit_base_patch16_224_learnablemasking_ratio_0.95_epoch_800/example-finetune.sh (I think the scripts for kinetics and ssv2 were put in the wrong folder), with the only difference being I'm using 32 gpus with batch size=16 for each GPU instead of 64 gpus with batch size=8 for each GPU. However, I only got 68.7 top 1 accuracy. I noticed in the paper that ssv2 fine-tuning base learning rate is 1.5e-4 instead of 5e-4 that is shown in the script. Could you confirm the configurations for fine-tuning ssv2? Thanks in advance.

**Comment by wgcban** [comment]
Hi @wren93, Thanks for pointing out the mistake. I can confirm that we used base learning rate of ``5e-4`` for the finetuning on ssv2 dataset instead of ``1.5e-4``.

**Comment by wren93** [comment]
Hi @wgcban thank you for the answer. To clarify, does that mean I should use learning rate=5e-4 x 512 / 256 = 1e-3 if I use batch size=512? Or should I just use 5e-4 for batch size=512 as shown in the script?

**Comment by wren93** [comment]
nevermind, it has been calculated in the code

**Comment by wgcban** [comment]
@wren93 yes.
